cmake/RobotDARTConfigVersion.cmake.in

```diff
@@ -0,0 +1,11 @@
+set(PACKAGE_VERSION "@robot_dart_VERSION@")
```

Comment:
Is this file used in any way? I guess we should use it to tell to CMake which version of RobotDART we want. I think at the moment it's not being used.. @jbmouret how were you planning on using it?